docs/chain-details/govprocess.md

```diff
@@ -0,0 +1,66 @@
+# The Proposal Process
+
+*The governance process is divided into a few steps that are outlined below*:
+ 
+### 1. The Idea
+Governance proposals potentially impact many stakeholders. It is suggested to introduce your idea with known community members before investing resources into drafting a proposal to get the very initial feedback. If you know someone who is very experienced in the topic, you may try to communicate with them about a concise overview of what you think will result from proposed changes.
+
+### 2. Begin your drafted proposal 
+After you confirm the idea, you may start to draft and describe some potential elements of the proposal. It is recommended to also consider the anticipated questions that the community will likely ask. 
+
+Please ensure to include the following information in your proposal: 
+- Overview of this proposal
+- Introduction of team member(s)
+- The problem/issue this project solve/The reason for creating this proposal
+- The details of the solution 
+- The impact brings to the chain of the proposal
+- Timeline, if applicable
+- How can the quality of the deliverables be assessed, if applicable.
+&nbsp;
+
+Proposal Types:
+You may choose to put your proposal in any of the following formats:
```

Comment:
Perhaps also `CommunityPoolSpendProposal` for things such as https://github.com/crypto-org-chain/chain-main/discussions/525 ? @calvinaco @lezzokafka

Currently, the community pool doesn't much in, but it can funded for promising proposals and the final funding decision will be done by the onchain vote?